files/templates/alertmanager/alertmanager.yaml

```diff
@@ -6,487 +6,487 @@ global:
 [[- end ]]
   slack_api_url: [[ .SlackApiURL ]]
```

Comment:
Should this file not be gone?

Reply:
Well, I had conflicts in the usage of configSecret and the `alertmanagerConfiguration`. 
they couldn't be loaded without minimum 1 route.

My idea is to keep some global config in the `configSecret` for now, 
Later, once `alertmanagerConfiguration` has more features so we can fully remove the configSecret.

so, configSecret defines global config.

and the `AlertmanagerConfig` gets merged into the config and overwrite the `root` route.